docs/api/graphql_operations.md

```diff
@@ -0,0 +1,209 @@
+# GraphQL operations
```

Comment:
Why "operations" ? It doesn't really have a meaning in the GraphQL domain. What about "mutations" ? In that case, they're specifically "content mutations"

Reply:
I picked "operations", because I did find it used in GraphQL doc (cf. https://graphql.org/learn/queries/#operation-name), and I think "mutations" may not be intuitive for someone who is not very familiar with GraphQL - unless we can assume it should be?